prow/scripts/cluster-integration/kyma-gke-end-to-end-test.sh

```diff
@@ -0,0 +1,238 @@
+        discoverUnsetVar=true
+    fi
+done
+if [ "${discoverUnsetVar}" = true ] ; then
+    exit 1
+fi
+
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+export KYMA_SOURCES_DIR="${KYMA_PROJECT_DIR}/kyma"
+export KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
+
+### For provision-gke-cluster.sh
+export GCLOUD_PROJECT_NAME="${CLOUDSDK_CORE_PROJECT}"
+export GCLOUD_COMPUTE_ZONE="${CLOUDSDK_COMPUTE_ZONE}"
+export GCLOUD_SERVICE_KEY_PATH="${GOOGLE_APPLICATION_CREDENTIALS}"
+
+### For generate-cluster-backup-config.sh
+export BACKUP_CREDENTIALS="${KYMA_BACKUP_CREDENTIALS}"
+export BACKUP_RESTORE_BUCKET="${KYMA_BACKUP_RESTORE_BUCKET}"
+
+readonly REPO_OWNER=$(echo "${REPO_OWNER_GIT}" | tr "[:upper:]" "[:lower:]")
+readonly REPO_NAME_GIT=$(echo "${REPO_NAME_GIT}" | tr "[:upper:]" "[:lower:]")
+readonly CURRENT_TIMESTAMP=$(date +%Y%m%d)
+
+readonly STANDARIZED_NAME=$(echo "${INPUT_CLUSTER_NAME}" | tr "[:upper:]" "[:lower:]")
+readonly DNS_SUBDOMAIN="${STANDARIZED_NAME}"
+export CLUSTER_NAME="${STANDARIZED_NAME}"
+
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/library.sh"
+
+removeCluster() {
+    #Turn off exit-on-error so that next step is executed even if previous one fails.
+    set +e
+
+    EXIT_STATUS=$?
+
+    shout "Fetching OLD_TIMESTAMP from cluster to be deleted"
+	readonly OLD_TIMESTAMP=$(gcloud container clusters describe "${CLUSTER_NAME}" --zone="${GCLOUD_COMPUTE_ZONE}" --project="${GCLOUD_PROJECT_NAME}" --format=json | jq --raw-output '.resourceLabels."created-at"')
+
+
+    shout "Deprovision cluster: \"${CLUSTER_NAME}\""
+    date
+
+    #save disk names while the cluster still exists to remove them later
+    DISKS=$(kubectl get pvc --all-namespaces -o jsonpath="{.items[*].spec.volumeName}" | xargs -n1 echo)
+    export DISKS
+
+    #Delete cluster
+    shout "Delete cluster $CLUSTER_NAME"
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/deprovision-gke-cluster.sh
+    TMP_STATUS=$?
+    if [[ ${TMP_STATUS} -ne 0 ]]; then EXIT_STATUS=${TMP_STATUS}; fi
+
+
+    if [ -v "${GATEWAY_IP_ADDRESS}" ] && [ -v "${GATEWAY_DNS_FULL_NAME}" ]; then
+        shout "Delete Gateway DNS Record"
+        date
+        IP_ADDRESS=${GATEWAY_IP_ADDRESS} DNS_FULL_NAME=${GATEWAY_DNS_FULL_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/delete-dns-record.sh
+        TMP_STATUS=$?
+        if [[ ${TMP_STATUS} -ne 0 ]]; then EXIT_STATUS=${TMP_STATUS}; fi
+    fi
+
+
+    if [ -v "${GATEWAY_IP_ADDRESS_NAME}" ]; then
+        shout "Release Gateway IP Address"
+        date
+        IP_ADDRESS_NAME=${GATEWAY_IP_ADDRESS_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/release-ip-address.sh
+        TMP_STATUS=$?
+        if [[ ${TMP_STATUS} -ne 0 ]]; then EXIT_STATUS=${TMP_STATUS}; fi
+    fi
+
+    if [ -v "${REMOTEENVS_IP_ADDRESS}" ] && [ -v "${REMOTEENVS_DNS_FULL_NAME}" ]; then
```

Comment:
`./prow/scripts/cluster-integration/kyma-gke-end-to-end-test.sh: line 84: [: -v: unary operator expected`